website/blog/2023-03-29-release-0.13.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Release Notes - Podman Desktop 0.13
+description: Podman Desktop 0.13 has been released! 
+slug: podman-desktop-release-0.13
+authors: [deboer]
+tags: [podman-desktop, release, kubernetes, compose]
+hide_table_of_contents: false
+---
+
+import ReactPlayer from 'react-player'
+
+This release note covers Podman Desktop 0.13 release changes.
+
+<!--Main Features-->
+
+- **Podman Version**: Podman 4.4.4 now included in Windows and Mac installers.
+- **Compose**: Support for installing Podman Compose.
+- **Extensions**: Improved support for extending Podman Desktop with additional capabilities.
+- **UX and UI Improvements**: Welcome page, task manager, resources, and update alerts.
+
+Podman Desktop 0.13 is now available. [Click here to download it](/downloads)!
+
+![podman-desktop-0-13-hero](img/podman-desktop-release-0.13/podman-desktop-release-0.13.png)
+
+<!--truncate-->
+_________________
+
+## Release Details
+
+### Update to Podman v4.4.4
+
+Podman Desktop 0.13 embeds [Podman 4.4.4](https://github.com/containers/podman/releases/tag/v4.4.4) in
+Windows and macOS installers [#1456](https://github.com/containers/podman-desktop/pull/1456). Make sure to upgrade to benefit from the latest Podman features and bug fixes.
+
+### Compose Support
+
+We've added support for Podman Compose [#1578](https://github.com/containers/podman-desktop/pull/1578)
+including an easy button to install it.
+
+### Extensions
+
+We've published the Podman Desktop Extension API to npmjs [#1727](https://github.com/containers/podman-desktop/pull/1727),
+documented [how to create, publish, and install your own extensions](https://podman-desktop.io/docs/extensions), and
+expanded the API for QuickPickInput [#1839](https://github.com/containers/podman-desktop/pull/1839).
```

Comment:
running sentence, maybe good to put these three in bullet points instead?

Reply:
Switched to bullets in second commit.